cft_sidecar.yaml

```diff
@@ -46,6 +58,18 @@ Parameters:
     Default: false
     ConstraintDescription: must specify 'true' or 'false'.
 
+  SidecarTLSCertificateRoleArn:
+    Description: "(Optional) ARN of an AWS IAM Role to assume when reading the TLS certificate."
+    Type: String
+    Default: ""
+    AllowedPattern: "^(|arn:.+)$"
+
+  SidecarCACertificateRoleArn:
+    Description: "(Optional) ARN of an AWS IAM Role to assume when reading the CA certificate."
+    Type: String
+    Default: ""
+    AllowedPattern: "^(|arn:.+)$"
+
```

Comment:
These parameters are optional at template level, but the script will fail. Please adjust the script to add an info message to the initialization logs when the secrets are not provided. The message should point that a self-signed certificate will be created as the secret containing the certificate was not provided.